### DISCUSSION TITLE
Is `ARG` not OCI Spec-compliant keyword? If not what is the alternative?

### DISCUSSION THREAD

**Discussion by sreevatsank1999** [containers/podman#22093] [Q&A]
While looking up the error, I found this issue #8477, In connection with that

Is `ARG` not an OCI Spec-compliant keyword? And why does the Warning say "`SHELL` is not supported" and not `ARG`?

>STEP 2/14: ARG DOCKER_LANG=en_US
>WARN[0022] SHELL is not supported for OCI image format, [/bin/bash -c] will be ignored. Must use `docker` format
>--> eea190125975
>STEP 3/14: ARG DOCKER_TIMEZONE=America/Los_Angeles
>WARN[0022] SHELL is not supported for OCI image format, [/bin/bash -c] will be ignored. Must use `docker` format
>--> cec389f70d05

Is this a bug?

Update: 
I am not sure what is up with this. I am getting this warning for every build step that is not used from cache
>WARN[0002] SHELL is not supported for OCI image format, [/bin/bash -c] will be ignored. Must use `docker` format
--> 8f779e04c183
STEP 9/17: RUN passwd -d ${USERNAME}
passwd: password changed.
WARN[0004] SHELL is not supported for OCI image format, [/bin/bash -c] will be ignored. Must use `docker` format
--> 33f8ad1e1385
STEP 10/17: RUN echo "$DOCKER_TIMEZONE" > /etc/timezone &&     ln -s -f /usr/share/zoneinfo/$DOCKER_TIMEZONE /etc/localtime
WARN[0006] SHELL is not supported for OCI image format, [/bin/bash -c] will be ignored. Must use `docker` format
--> 5474c60ba938
STEP 11/17: ENV USER ${USERNAME}
WARN[0006] SHELL is not supported for OCI image format, [/bin/bash -c] will be ignored. Must use `docker` format
--> 504f7fc1f644
STEP 12/17: ENV HOME /home/${USERNAME}
WARN[0006] SHELL is not supported for OCI image format, [/bin/bash -c] will be ignored. Must use `docker` format
--> 96b0e8d4b193
STEP 13/17: WORKDIR /home/${USERNAME}
WARN[0006] SHELL is not supported for OCI image format, [/bin/bash -c] will be ignored. Must use `docker` format
--> 06e9efea257e
STEP 14/17: USER ${USERNAME}
WARN[0006] SHELL is not supported for OCI image format, [/bin/bash -c] will be ignored. Must use `docker` format
--> ae98fe46aa2a
STEP 15/17: RUN echo "Hello from fplll-devel"
Hello from fplll-devel
WARN[0007] SHELL is not supported for OCI image format, [/bin/bash -c] will be ignored. Must use `docker` format
--> 3e2149355036
STEP 16/17: ENTRYPOINT [  ]
WARN[0007] SHELL is not supported for OCI image format, [/bin/bash -c] will be ignored. Must use `docker` format
--> 8891646c6728
STEP 17/17: CMD [ "tail", "-f", "/dev/null" ]
COMMIT latred:latest
WARN[0008] SHELL is not supported for OCI image format, [/bin/bash -c] will be ignored. Must use `docker` format
--> 9a77674e2215
Successfully tagged localhost/latred:latest
9a77674e2215da1a63dd5699c921b41ac3b9ddc813b8936a1e6f17b99d46957e
exit code: 0

**Comment by rhatdan** [comment]
SHELL is only allowed in Docker format, the OCI format which Podman build and Buildah defaults to does not support SHELL option.

If you add --format docker to your podman build command, it should work.
